### ISSUE TITLE
Part Name - trapezium

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#2612]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Goal**
Get all the bone parts into Arctos.

**Context**
It's a bone.

**Table**
http://test.arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_part_name

**Value**
trapezium

**Definition**
A carpal bone in the hand. It forms the radial border of the carpal tunnel. Also known as the  greater multangular bone, it is homologous with the first distal carpal of reptiles and amphibians. https://en.wikipedia.org/wiki/Trapezium_(bone)

**Collection type**
Mamm, ES

**Part tissue flag**
yes

**Priority**
Please assign a priority-label.

**Comment by campmlc** [comment]
There are 206 bones in the human skeleton, and others in other taxa . . .
.have we decided to name each one separately? Seems like that is a bigger
question.

**Comment by dustymc** [comment]
>decided to name each one separately

Yes, agreed, we should have a high-level discussion before creating every structure and [lump](https://github.com/ArctosDB/arctos/issues/2585) as a part.

**Comment by Jegelewicz** [comment]
Paleo people often have only a single bone. I am open to the discussion, but I don't want it to drag on for months while I wait to catalog stuff...

**Comment by campmlc** [comment]
If we go this route, it would be preferable to find an external source to
add all possible values, rather than one bone at a time. . .
If the part were "bone, entire" or "bone, fragment", or at least "bone,
appendicular skeleton", "bone, cranial", "bone, vertebral", "bone, pelvic"
or some such categorization etc that would at least simplify and make
searching easier.  The specific bone name could then be a part attribute?

**Comment by ewommack** [comment]
That might be a good idea. 
Think if we had to name all of the bones in the fish skeleton....and they keep evolving and changing them!

**Comment by dustymc** [comment]
I think that's the high-level discussion - should bumps and "insignificant" bones and such be parts, or in some secondary structure?


* big lists are hard to browse; given "skull" in a reasonably-sized list users will find what they want, given 20-some-odd bones plus "skull," then repeat for metacarpals and fish-bits and whatever grasshoppers do and .... to build a huge list and users will get overwhelmed and fail to find what they want.
* we know "trapezium," some users probably want that and nothing else, pushing it back would make things more difficult for them.

Etc. I think there are lots of compelling arguments going both ways. The AWG exists to answer hard questions like this....

**Comment by ewommack** [comment]
I can put it on the up coming Agenda. Is there a label we use that can help me remember or spot issues like this that we'd really like to bring to the AWG?

**Comment by dustymc** [comment]
>label

I use the project - I'm not sure it works, but that's what I do anyway!

<img width="509" alt="Screen Shot 2020-04-24 at 4 33 40 PM" src="https://user-images.githubusercontent.com/5720791/80264558-6c8aed00-8649-11ea-8aaf-eabd208dd57c.png">

**Comment by ewommack** [comment]
Thanks @dustymc! I'll make sure to check that project out when I start to put together the agenda.

**Comment by Jegelewicz** [comment]
@ewommack my vision was to have that project BE the agenda for issues meetings....

**Comment by ewommack** [comment]
@Jegelewicz You want it for the issues meeting and not the working group?

**Comment by Jegelewicz** [comment]
@ewommack ideally it would be for both, but some of the things we cover in the non-issues meeting never really get discussed in GitHub (in my perfect world, they would).

**Comment by dustymc** [comment]
I think this need the widest possible coverage - having many more parts would likely significantly change some things, require some UI adjustments, etc.

**Comment by ewommack** [comment]
Ok I made sure it is on each agenda. They're in the google drive shared folder if you want to add something else.
